flixel/group/FlxSpriteGroup.hx

```diff
@@ -1172,6 +1172,16 @@ class FlxTypedSpriteGroup<T:FlxSprite> extends FlxSprite
 		return Frames;
 	}
 
+	override function get_graphicWidth():Float
+	{
+		return length == 0 ? width : findMaxXHelper() - findMinXHelper();
```

Comment:
findMaxX() uses the hitbox, we'll need a new findMaxGraphicX() for this

Reply:
sorry for late reply (I've been busy), but yeah, I'll do that